defaults/main.yml

```diff
@@ -77,6 +78,10 @@ k8s_cluster_join_configuration:
       caCertHashes:
         - "sha256:{{ k8s_cluster_root_ca_hash }}"
       unsafeSkipCAVerification: false
+  controlPlane:
```

Comment:
I suppose it's better to avoid using this option in default configuration. I guess we will join workers to the cluster more frequently then masters